sys-firmware/amd-microcode/amd_microcode-20240710.recipe

```diff
@@ -34,4 +38,9 @@ INSTALL()
 	mkdir -p $dataDir/firmware/amd-ucode
 	cp $sourceDir/* $sourceDir2/* $sourceDir3/* $sourceDir4/* $sourceDir5/* \
 		$dataDir/firmware/amd-ucode/
+
+	# install postinstallscript
+	mkdir -p $postInstallDir
+	echo "\$portDir = \"$portDir\""
```

Comment:
NB: I suppose that https://github.com/haikuports/haikuporter/issues/221 is still not fixed ... (but it isn't relevant here anyway because INSTALL runs in the chroot)